src/store/mutations.ts

```diff
@@ -191,6 +199,9 @@ export default {
    * update pipeline.
    */
   setPipeline(state: VQBState, { pipeline }: { pipeline: Pipeline }) {
+    if (state.currentPipelineName == null) {
```

Comment:
It seems that `currentPipelineName` can't be `null` from a type point of view so I guess 

```suggestion
    if (state.currentPipelineName === undefined) {
```

would be better or am I missing something?